assets/src/adaptivity/rules-engine.ts

```diff
@@ -66,11 +66,11 @@ const applyToEveryCondition = (top: TopLevelCondition | NestedCondition, callbac
   });
 };
 
-const evaluateValueExpression = (value: string, env: Environment) => {
+const evaluateValueExpression = (value: string, env: Environment, key?: string) => {
```

Comment:
These changes were already approved as part of https://github.com/Simon-Initiative/oli-torus/pull/4309